src/docs/components/DocsActiveBlock.jsx

```diff
@@ -31,9 +31,11 @@ class DocsActiveBlock extends Component {
         }, {key: 'component'});
     }
 
+
     this.state = {
       showCode: null,
       changeBackground: this.props.backgroundColor || BACKGROUND_COLOR.LIGHT,
+      centeredItems: typeof this.props.centeredItems === 'undefined' ? true : this.props.centeredItems,
```

Comment:
Do we need state, not props?
When you will be using this props You could use default value ({centeredItems=true})